docs/journalist.rst

```diff
@@ -1,289 +1,299 @@
+Browser, ideally in a new Tails session for highly sensitive
+submissions. For more information, visit the Tails guide to `working
+with sensitive documents`_.
+
+We recommend always doing as much work as possible inside of Tails
+before copying documents back to your *Journalist Workstation*. This
+includes stripping metadata with MAT.
+
+When you no longer need documents, you can right-click on them and
+choose **Wipe** to delete them.
 
 |Wiping documents|
 
-Encrypt and move documents to Journalist Workstation
-----------------------------------------------------
+As long as you are using the latest version of Tails, you should be
+able to open any submitted documents without the risk of malicious
+files compromising the *Secure Viewing Station*. However, even if a
+compromise does occur, Tails is designed so that the next time you
+reboot, the malware will be gone.
 
-Before you move documents back to the ``Transfer Device`` to copy them
-to your workstation you should encrypt them to your personal GPG public
-key that you imported when setting up the *Secure Viewing Station* to
-begin with.
+.. _`working with sensitive documents`: https://tails.boum.org/doc/sensitive_documents/index.en.html
+.. _`Metadata Anonymisation Toolkit`: https://mat.boum.org/
 
-Right-click on the document you want to encrypt and choose
-``Encrypt...``
+Encrypting and Moving Documents to the *Journalist Workstation*
+---------------------------------------------------------------
+
+Before moving documents back to the *Transfer Device* to copy them to
+your workstation, encrypt them to your personal GPG key that you
+imported when setting up the *Secure Viewing Station*.
+
+To do this, right-click on the document you want to encrypt and choose
+**Encrypt...**.
 
 |Encrypting 1|
 
-Then choose the public keys of the journalist you want to encrypt the
-documents to and click ``OK``.
+Then choose your public key (and, if you choose, any additional keys,
+such as an editor's) and click **OK**.
 
 |Encrypting 2|
 
-When you are done you will have another document with the same filename
-but ending in .gpg that is encrypted to the GPG keys you selected. You
-can copy the encrypted documents to the ``Transfer Device`` to transfer
-them to your workstation.
+When you are done encrypting, you will have another document with the
+same filename but ending in ".gpg". This file is encrypted to the GPG
+keys you selected. You can safely copy these encrypted files to the
+*Transfer Device* to transfer them to your workstation.
 
 |Encrypted document|
 
-Decrypt and prepare to publish
-------------------------------
+Decrypting and Preparing to Publish
+-----------------------------------
 
-Plug the ``Transfer Device`` into your workstation computer and copy the
-encrypted documents to it. Decrypt them with ``gnupg``.
+Plug the *Transfer Device* into your workstation computer and copy
+over the encrypted documents. Decrypt them with GnuPG.
```

Comment:
We've used the term "GPG" throughout, so let's stick with that here instead of "GnuPG"